imports/ui/PlayerRoom.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Presence } from "meteor/tmeasday:presence";
+import { GamesCollection } from "../api/games";
+import { useTracker } from "meteor/react-meteor-data";
+
+// put the player into the gameID (game room) after pressing play(done)
+// dont show the list for both playing players(done)
+// dont show users if they are in the game for other players(done)
+// make a quit game button ant after click set status back to isPlaying: false(done)
+// and show them the waiting room(done)
+
+const Room = ({ room }) => {
+  const myName = Presence.state("user");
+
+  const [hideList, setHideList] = useState(false);
+  const [quit, setQuit] = useState(false);
+
+  const game = useTracker(() => {
+    Meteor.subscribe("games");
+    const currentGame = GamesCollection.findOne(Session.get("gameID"));
+    if (currentGame && currentGame.players && currentGame.players.length > 1) {
+      setHideList(true);
+      if (quit === false) {
+        setQuit(false);
+        Meteor.call("userPresence", myName.user, (err, result) => {
+          Session.set({
+            user: myName.user,
+            isPlaying: true,
+          });
+        });
+      } else {
+        setQuit(true);
+        setHideList(false);
+      }
+    }
```

Comment:
This is rather confusing logic, also unnecessary `setQuit(false)` the fact that you are using `quit` from the state already implies that it's has this value, something like this should work as well. Also you can use optional chaining to avoid multiple `&&
```suggestion
    if (currentGame?.players?.length > 1 && !quit) {
      setHideList(true);
      Meteor.call("userPresence", myName.user, (err, result) => {
        Session.set({
          user: myName.user,
          isPlaying: true,
        });
      });
    }
```